terraform/environment/modules/environment/rds.tf

```diff
@@ -57,7 +57,8 @@ resource "aws_db_instance" "api" {
 
   lifecycle {
     ignore_changes = [
-      latest_restorable_time
+      latest_restorable_time,
+      snapshot_identifier
```

Comment:
What's the purpose of this extra variable?

Reply:
snapshot identifier always changes on a re-apply, so we want to avoid that as it rebuilds the DB instance.